tavern/internal/www/src/pages/quest-list/quest-table/QuestTable.tsx

```diff
@@ -88,37 +101,69 @@ export const QuestTable = (props: Props) => {
                 );
             },
             footer: (props:any) => props.column.id,
+            sortingFn: "alphanumeric"
         },
         {
             id: "inprogress",
             header: 'In progress tasks',
             accessorKey: "inprogress",
             footer: props => props.column.id,
+            sortingFn: "alphanumeric"
         },      
         {
             id: "queued",
             header: 'Queued tasks',
             accessorKey: "queued",
             footer: props => props.column.id,
+            sortingFn: "alphanumeric"
         },  
-        // {
```

Comment:
Dead code?

Reply:
Yes I had this for reference but we went the route of having output details in a drawer.